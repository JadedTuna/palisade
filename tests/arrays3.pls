in {
    high arg: bool;
}
out {
    low arr[3]: bool;
}

x[3] := [false, true, arg];
// error, must specify size for z
z := x;
