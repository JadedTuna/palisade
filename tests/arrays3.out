[1;31merror: [0marray definition must have size specification
   8 | x[3] := [false, true, arg];
   9 | // error, must specify size for z
  10 | z [1;31m:=[0m x;
       [1;31m~~[0m[1;31m^^[0m
